kernel/aster-nix/src/vm/vmar/vm_mapping.rs

```diff
@@ -24,6 +24,8 @@ pub struct VmMapping {
     parent: Weak<Vmar_>,
     /// The mapped vmo. The mapped vmo is with dynamic capability.
     vmo: Vmo<Rights>,
+    /// The mapping is shared among processes
+    is_shared: bool,
```

Comment:
-> Whether the mapping is shared among processes

The `is_shared` in `VmMapping` can also indicate that modifications to the mapping in the case of file-backed scenarios will be reflected in the underlying file. I think we can add a TODO to the doc here to reflect this situation and indicate we do not support it yet.